src/runtime/Makefile

```diff
@@ -261,6 +265,10 @@ DEFGUESTPREATTESTATIONSECRETTYPE ?= connection
 DEFSEVCERTCHAIN ?= /opt/sev/cert_chain.cert
 DEFSEVGUESTPOLICY ?= 0
 
+# SNP
+DEFSNPGUEST ?= true
+DEFSNPGUESTPOLICY ?= 0x30000
```

Comment:
Can we have a comment about that?

Reply:
Will add more comments for sure in this batch of changes. 
I got it from [#5796](https://github.com/kata-containers/kata-containers/pull/5796/files) 

Niteesh's explanation was:
"
Refer SEV Secure Nested Paging Firmware ABI Specification (sectin 4.3) to change this value.
unspecified or == 0  -->  0x30000 i.e. Bit#17 is '1' and Bit#16 is '1' (SMT is allowed)
"
But that is a very good point.